tests/phpunit/tests/rest-api/rest-sidebars-controller.php

```diff
@@ -128,6 +141,8 @@ public function test_context_param() {
 	 * @ticket 41683
 	 */
 	public function test_get_items() {
+		wp_widgets_init();
```

Comment:
Should this go in `setUp()` so that we can avoid some repetition?

Reply:
Explored doing that. But it skews all the other tests that don't have it. In reviewing other widget tests, this init function is placed in each test that needs it. Not sure why yet. But it's on my test review TODO list across the test suite.